### PR TITLE
Fix Falcon2 and 3 compatibility issue

### PR DESCRIPTION
**Pull request by mortymacs**

### CODE REVIEW

File: spectree/plugins/falcon_plugin.py [hunk 1]
```diff
@@ -31,11 +31,15 @@ def on_get(self, req, resp):
 class FalconPlugin(BasePlugin):
     def __init__(self, spectree):
         super().__init__(spectree)
-        from falcon import HTTPUnsupportedMediaType, MediaNotFoundError
+        import falcon
+
+        if falcon.__version__[0] == "3":
+            from falcon import MediaNotFoundError as UnsupportedMediaType
+        else:
+            from falcon import HTTPUnsupportedMediaType as UnsupportedMediaType
```
Comment (kemingy) [inline spectree/plugins/falcon_plugin.py:39]:
```suggestion
        try:
            from falcon import HTTPUnsupportedMediaType as UnsupportedMediaType
        except ImportError:
            from falcon import MediaNotFoundErroras UnsupportedMediaType
```

Comment (mortymacs) [reply spectree/plugins/falcon_plugin.py]: Done.

**Comment by mortymacs** [comment]
Thanks! Do you have any plan to have a release soon?

**Comment by kemingy** [comment]
> Thanks! Do you have any plan to have a release soon?

I have another problem with falcon 3. It generates a lot of redundant endpoints. Will release a new version after I fixed that one.

**Comment by mortymacs** [comment]
> > Thanks! Do you have any plan to have a release soon?
> 
> I have another problem with falcon 3. It generates a lot of redundant endpoints. Will release a new version after I fixed that one.

Do we have any tickets for it?

**Comment by kemingy** [comment]
> 
> 
> > > Thanks! Do you have any plan to have a release soon?
> > 
> > 
> > I have another problem with falcon 3. It generates a lot of redundant endpoints. Will release a new version after I fixed that one.
> 
> Do we have any tickets for it?

Check #139

**Pull request merged**